internal/controllers/aggregation/slice.go

```diff
@@ -98,8 +98,8 @@ func (s *sliceController) Reconcile(ctx context.Context, req ctrl.Request) (ctrl
 
 		if synthed := comp.Status.CurrentSynthesis.Synthesized; synthed != nil {
 			latency := maxReadyTime.Sub(synthed.Time)
-			if latency.Milliseconds() > 0 {
-				logger.V(0).Info("composition became ready", "latency", latency.Milliseconds())
+			if latency.Microseconds() > 0 {
+				logger.V(0).Info("composition became ready", "latency", latency.Abs().Microseconds())
```

Comment:
Any particular reason for switching to microseconds? Milliseconds is more... intuitive or at least common IMO.

Also- I'm curious what `Abs` does in this case, since we're already checking for positive.

Reply:
Fixed it so that we only use milliseconds